### ISSUE TITLE
Tick on some Hub items is not explained

### ISSUE THREAD

**Issue by itowlson** [fermyon/developer#1094]
(at least not that I managed to find)

Some Hub cards have a little tick in their "category" box:

![image](https://github.com/fermyon/developer/assets/865538/f1da61f6-9718-4536-b338-7ccc3d780966)

I thought I could find out what it was by hovering for a tooltip, but when I tried it slid up out of view.

**Comment by karthik2804** [comment]
It represents by Fermyon or verified by Fermyon.

**Comment by itowlson** [comment]
How does a reader know this though?  That's what the issue seeks to clarify.

**Comment by karthik2804** [comment]
Good point! I will look to add a tooltip explaining it.

**Comment by karthik2804** [comment]
I was looking at this again, we cannot add a tooltip on it as that slides away when we hover over the card. @flynnduism any thoughts on where we can establish what the checkmark means?

**Comment by VamshiReddy02** [comment]
I have a small suggestion, what do you think about adding a tooltip inside the card? For example:

<img width="372" alt="Screenshot 2024-01-13 at 9 08 34 PM" src="https://github.com/fermyon/developer/assets/99033623/e3b61b3d-8229-4ff7-8088-7545516fecc3">

**Comment by karthik2804** [comment]
Yes, we could do something like that inside the preview modal, but the issue remains on the card list where you cannot hover. we can probably redesign the checkmark into a small chip with the icon and "verified" as text to the logo. 

cc: @flynnduism
